### DISCUSSION TITLE
Problems with refine-mui

### DISCUSSION THREAD

**Discussion by MarkoLumbar** [refinedev/refine#5633] [Q&A]
Hello,

I'm trying to use the old version of refine, i downloaded it with npm create refine-app@latest -- --branch v3 but now i have problems with refine-mui, probably dependency of mui changed but now when I do npm run dev, i get this errors, probably i have to use refine-mui version that is compatible with refine version 3 but i cant seem to find what version thats it. Thank you for the help.

Errors:export 'default' (reexported as 'generateUtilityClass') was not found in '@mui/base/generateUtilityClass' (possible exports: generateUtilityClass, isGlobalState)
ERROR in ./node_modules/@pankod/refine-mui/dist/esm/index.js 3448:0-4185
export 'default' (reexported as 'generateUtilityClass') was not found in '@mui/base/generateUtilityClass' (possible exports: generateUtilityClass, isGlobalState)

ERROR in ./node_modules/@pankod/refine-mui/dist/esm/index.js 3448:0-4185
export 'default' (reexported as 'generateUtilityClasses') was not found in '@mui/base/generateUtilityClasses' (possible exports: generateUtilityClasses)

ERROR in ./node_modules/@pankod/refine-mui/dist/esm/index.js 3448:0-4185
export 'default' (reexported as 'unstable_composeClasses') was not found in '@mui/base/composeClasses' (possible exports: unstable_composeClasses)

**Comment by BatuhanW** [comment]
Hey @marlum732 you can use these versions:

```
        "@emotion/react": "^11.8.2",
        "@emotion/styled": "^11.8.1",
        "@mui/icons-material": "^5.8.3",
        "@mui/lab": "^5.0.0-alpha.85",
        "@mui/material": "^5.8.6",
        "@mui/x-data-grid": "^5.12.1",
```
